docs/finality-provider-operation.md

```diff
@@ -400,6 +400,13 @@ KeyDirectory = <path> # The `--home` path to the directory where the keyring is
 > instructions on setting up a Babylon node 
 > [here](https://github.com/babylonlabs-io/networks/tree/main/bbn-test-5/babylon-node/README.md).
 
+> ⚠️ **Critical RPC Configuration**:
+> When configuring your finality provider to a Babylon RPC node, you should
+> connect to a **single** node directly. Additionally you **must** 
+> ensure that this node has transaction indexing enabled (`indexer = "kv"`)
+> otherwise you will get a `duplicate vote error`. Using multiple RPC nodes or 
```

Comment:
Let's remove `otherwise you will get a duplicate vote error` as I'm not sure it's the only consequence.